minique/api.py

```diff
@@ -48,9 +48,15 @@ def enqueue(
     return job
 
 
-def get_job(redis: "Redis[bytes]", job_id: str) -> Job:
+def get_job(
+    redis: "Redis[bytes]",
+    job_id: str,
+    dequeue: bool = False,
+) -> Job:
     job = Job(redis, job_id)
     job.ensure_exists()
+    if dequeue:
+        redis.lrem(job.get_queue().redis_key, 0, job.id)
```

Comment:
I wonder if this should just be a function on `Job` that returns `true` if it did a thing and `false` if it didn't?